lib/ASTGen/Sources/ASTGen/Macros.swift

```diff
@@ -407,6 +407,29 @@ func checkMacroDefinition(
   }
 }
 
+// Make an expansion result for '@_cdecl' function caller.
+func makeExpansionOutputResult(
```

Comment:
Doesn't have to be this PR, but could this use `BridgedString` instead?

Reply:
`withBridgedString` can't escape the closure argument. Since we need to send the result back to C++ somehow, and currently (afaik) we don't have a way to call C++ lambda from Swift, I don't think it's super simple to use it.

Of course we can create `BridgedString` using the allocated buffer instead, but I'm not sure we should do it. 🤔

Reply:
To be clear, I meant the latter + having `swift_ASTGen_expand*` return it instead of taking it as an output parameter. Is the concern that `BridgedString` is currently only ever used in the other direction at the moment? IMO having it as the return value + a comment mentioning the caller owns the underlying buffer is enough to me. But like I said, it shouldn't block this PR so we can talk about it later.